packages/automation/src/mapping.ts

```diff
@@ -0,0 +1,60 @@
+import ethers from 'ethers';
+import { AutomationTriggerType, CommandContractType, EthereumNetwork } from './types';
+
```

Comment:
What is a benefit here of having two mappings instead of one ?

I mean 
`{
  [EthereumNetwork.GOERLI]: {
    '0xd0ca9883e4918894dd517847eb3673d656ec9f2d': ['uint256', 'uint16', 'uint256'], // TODO:
  },
}`


I see possible loss of flexibility here if between implementations trigger format changes. Sure we probably can bypass it by having `CommandContractType.CloseCommand_v2` enum value but i do not feel like we have to.

Reply:
this is done to enable the encoding/decoding by type, I've just added those functions. and, in general, it helps the readability of this code

Reply:
So In case we will change encoding between versions, do you plan on having multiple enums ? Like CommandContractType.CloseCommand_v2  I think it will mess up clents a lot. 

The way it is now you cannot have 2 different encodings for two different commands (versions) if they share same CommandContractType

and I believe that is a problem. 

If you want to pass information about type you can modify my proposal to:

`{ [EthereumNetwork.GOERLI]: { '0xd0ca9883e4918894dd517847eb3673d656ec9f2d': {
triggersDataFormat: ['uint256', 'uint16', 'uint256'], // TODO: }, 
commandType: CommandContractType.CloseCommand}`